docs/user-guide/custom-themes.md

```diff
@@ -302,7 +316,13 @@ full search implementation to your theme.
 <h1 id="search">Search Results</h1>
 
 <form action="search.html">
-  <input name="q" id="mkdocs-search-query" type="text" class="search_input search-query ui-autocomplete-input" placeholder="Search the Docs" autocomplete="off">
+  <input
+      name="q"
+      id="mkdocs-search-query"
+      type="text"
+      class="search_input search-query ui-autocomplete-input"
+      placeholder="Search the Docs"
+      autocomplete="off">
```

Comment:
I agree. The docs should probably show the minimal amount required to get things working. But it is not clear what that is in this case. What should this look like?